content/docs/12-enterprise-version/03-deploying-palette-with-helm.md

```diff
@@ -0,0 +1,326 @@
+        # certificate: ""
+        # key: ""
+
+        annotations:
+        # AWS example
+        service.beta.kubernetes.io/aws-load-balancer-internal: "true"
+        service.beta.kubernetes.io/aws-load-balancer-backend-protocol: tcp
+        service.beta.kubernetes.io/aws-load-balancer-ssl-cert: "<ARN:ACM>"
+        service.beta.kubernetes.io/aws-load-balancer-ssl-ports: "https"
+        ingressStaticIP: ""
+    ```
+ 
+ 14. Download the kubeconfig file for the EKS cluster. Ensure you can interact with the target cluster. You can validate by issuing a `kubectl` command. For additional guidance, refer to the [kubeconfig file for an Amazon EKS](https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html) guide.
+
+
+
+15. Install the Helm Chart using the following command. Replace the path in the command to match your local path of the Palette Helm Chart.
+
+  <br />
+
+  ```shell
+  helm install palette /path/to/chart.tgz --file /path/to/values.yaml
+  ```
+
+16. Monitor the deployment using the command below. Palette is ready when the deployments in namespaces `cp-system`, `hubble-system`, `jet-system` , and `ui-system` reach the *Ready* state.
+
+    <br />
+
+    ```shell
+    kubectl get pods --all-namespaces --watch
+    ```
+
+17. Create a DNS record that is mapped to the Palette `ingress-nginx-controller` load balancer. You can use the following command to retrieve the load balancer IP address.
+
+    <br />
+
+    ```shell
+    kubectl get service ingress-nginx-controller --namespace nginx --output jsonpath='{.status.loadBalancer.ingress[0].hostname}'
+    ```
+
+You now have a self-hosted instance of Palette installed in a Kubernetes cluster.
+
+<br />
+
+<InfoBox>
+
+To upgrade Palette with a new Helm release, or to modify the values used in the installation, use the following command. 
+
+  <br />
+
+  ```shell
+  helm upgrade palette /path/to/chart.tgz --file /path/to/values.yaml
+  ```
+
+</InfoBox>
+
+</Tabs.TabPane> 
+
+
+</Tabs>
+
+# Validation
+
+You can validate that the installation of Palette is successful by visiting the custom domain you assigned to the parameter
+`env.rootDomain` in the **values.yaml**.
+
+<br />
+
+
+<WarningBox>
+
+If you notice that the pods in the `hubble-system` namespace are not initializing as expected, it might be due to a delay in adding the DNS records for the rootDomain. The workaround is to terminate all pods except the pods related to `mongo-db` in the `hubble-system` namespace to trigger a redeployment of the pods.
+
+  <br />
+
+  ```shell
+  kubectl delete pods --namespace hubble-system --selector=role!=mongo
+  ```
+
+</WarningBox>
+
+# Next Steps
+
+Start exploring the Palette system dashboard so that you become familiar with the available actions you can make as an administrator. Check out the [System Console Dashboard](/enterprise-version/system-console-dashboard) resource to learn more.
```

Comment:
```suggestion
Start exploring the Palette system dashboard so that you become familiar with the available actions you can take as an administrator. Check out the [System Console Dashboard](/enterprise-version/system-console-dashboard) resource to learn more.
```